app/index.js

```diff
@@ -32,7 +32,10 @@ var app = new Vue({
       return `${account.substr(0, 5)}...${account.substr(-3)}`;
     },
     imageFromIpfs: async function (uri) {
-      let url = uri.replace("ipfs://", "https://ipfs.io/ipfs/");
+      let url = uri;
```

Comment:
Looks like this code is extending `imageFromIpfs` method to get images via http protocal as well. So the naming of the method should be changed.